implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -76,6 +76,7 @@ where
         }
         .with_context(OpenTelemetryContext::current_context()),
     );
+    opts.force_flush();
```

Comment:
Will this block indefinitely until all traces are exported? Or all the processors we use implement the cutoff logic?

Reply:
Yes the cutoff logic applies all the time. It was an additional reason why I was not getting those user events.